### DISCUSSION TITLE
Discussion view behaviour has changed

### DISCUSSION THREAD

**Discussion by rdica** [jamulussoftware/jamulus#3102] [General]
At some point in the past few or more months something changed in how topics are displayed in Discussions. It used to be that when a new comment was added to a Discussion, the topic would bold, indicating a new comment, and sorting would place the topic at the top of the list. This is no longer the case, only the sorting seems to indicate a new comment, and if you didn't remember the previous sorting you wouldn't know what discussions were actually updated.

I couldn't find any settings for my account that would modify this behaviour. Is this a repo-level change or github-wide?

Is there a way to return to the previous behaviour?

**Comment by ann0see** [comment]
We didn't change anything as far as I remember. So it's probably a Github change.

**Reply by pljones** [reply]
I guess they've been spending Microsoft's money on various cosmetics...
